src/Compilers/VisualBasic/Portable/Operations/VisualBasicOperationFactory_Methods.vb

```diff
@@ -190,7 +190,7 @@ Namespace Microsoft.CodeAnalysis.Operations
                             If argument.Kind = BoundKind.ArrayCreation Then
                                 kind = ArgumentKind.ParamArray
                             End If
-                        Else
+                        ElseIf Not defaultArguments.IsNull AndAlso defaultArguments(index) Then
```

Comment:
>Not defaultArguments.IsNull AndAlso defaultArguments(index) [](start = 31, length = 59)

It looks like indexing is safe even for IsNull case. Consider simply passing the boolean instead of passing entire structure by ref.